Cargo.toml

```diff
@@ -36,3 +55,8 @@ sync = []
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
+
+# [workspace]
```

Comment:
Sorry I was trying to get the examples to work with VSCode rust plugin and added this.  It ended up causing other issues since I guess "examples" aren't workspaces?

Reply:
I found several other projects that do add examples to the workspace and it does improve the developer experience I think. Got it working based on another comment (https://github.com/containerd/ttrpc-rust/pull/172#discussion_r1115153595)

Reply:
This was causing the CI to fail since the files aren't there before cargo fmt was called.  I've removed it, I still think it is a good idea to get this working but can in separate PR.